tests/services/components/test_access_component.py

```diff
@@ -20,19 +20,23 @@
 from invenio_rdm_records.services.components import AccessComponent
 
 
-def test_access_component_valid(minimal_record, identity_simple, users):
-    record = RDMRecord.create(minimal_record)
+def test_access_component_valid(
+    minimal_record, parent, identity_simple, users
+):
+    record = RDMRecord.create(minimal_record, parent=parent)
     component = AccessComponent(RDMRecordService())
     component.create(identity_simple, minimal_record, record)
 
-    assert len(record.access.owners) > 0
+    assert len(record.parent.access.owners) > 0
 
 
+# TODO this test doesn't currently make sense (the parents will handle owners)
```

Comment:
these tests test functionality that has been moved to the currently unused `ParentRecordAccessComponent`, so i'm skipping them for now